### PR TITLE
Docs: adding typedoc

### PR DESCRIPTION
**Pull request by barnjamin**
Adding docs using the built-in package support to document all the subpackages into a single directory we can serve from GH pages

### CODE REVIEW

File: packages/solana/src/utils/wormhole/coder/idl.ts [hunk 1]
```diff
@@ -54,12 +54,6 @@ export class IdlCoder {
       case 'i128': {
         return borsh.i128(fieldName);
       }
-      case 'u256': {
```
Comment (kev1n-peters) [inline packages/solana/src/utils/wormhole/coder/idl.ts:57]: Why are we removing these? Are they not used anywhere?

File: packages/examples/dist/src/index.js
Comment (kev1n-peters) [inline packages/examples/dist/src/index.js:1]: I don't think we should add the files in the `dist` directory

File: packages/solana/src/utils/wormhole/coder/idl.ts (hunk 1 above)
Comment (barnjamin) [reply packages/solana/src/utils/wormhole/coder/idl.ts:57]: I don't think they're used and they puke when I try to build docs.

See https://github.com/nonergodic/sdkv2/pull/3#discussion_r1268194985

File: packages/examples/dist/src/index.js
Comment (barnjamin) [reply packages/examples/dist/src/index.js:1]: doh! removing

**Review by kev1n-peters — APPROVED** [review]

**Pull request merged**